lib/nerves/artifact.ex

```diff
@@ -33,11 +33,20 @@ defmodule Nerves.Artifact do
   @doc """
   Produces an artifact which can be fetched when calling `nerves.artifact.get`.
```

Comment:
The text and functions here are using the words "artifact" and "archive" interchangably. I believe that you wanted line 38 to read "mix archive" instead of "mix artifact", but you should probably review the lines around this comment to make sure that they're like you want.